scss/_patterns_article-pagination.scss

```diff
@@ -73,8 +74,8 @@
     &::before {
       @extend %chevron-icon;
 
+      transform: rotate(90deg);
```

Comment:
This causes linting errors as we sort the props alphabetically. Same with the transform added below.